README.md

```diff
@@ -34,6 +34,11 @@ Examples:
 
 (`host/managed` is for Foreman 1.2+, change to just `host` for Foreman 1.1)
 
+Plugins need to add the plugin name before the model name. For example:
+
+    ~/foreman/config/hooks/katello/product/before_create/01_integrate_catalog.sh
+    ~/foreman/config/hooks/foreman_tasks/task/dynflow_task/after_create/90_record_action.sh
```

Comment:
This example doesn't seem to work:

    Unable to render #<ForemanTasks::Task::DynflowTask:0x00000006b0fa80> (ForemanTasks::Task::DynflowTask) using RABL: Cannot find rabl template 'api/v2/dynflowtasks/show'

This is an STI model, so it actually needs to be looking up the template for 'tasks'?